apps/emqx_authn/src/emqx_authn_password_hashing.erl

```diff
@@ -71,10 +71,10 @@ fields(bcrypt_rw) ->
         [
             {salt_rounds,
                 sc(
-                    integer(),
+                    range(5, 10),
                     #{
-                        default => 10,
-                        example => 10,
+                        default => 5,
+                        example => 5,
                         desc => "Salt rounds for BCRYPT password generation."
```

Comment:
It's not just salt rounds, is it? Number of rounds encoded in salt is then used to hash the whole thing.

Reply:
Yes, the salt contains the number of rounds used for hashing.

Reply:
May be rephrase it as "Work factor for bcrypt password generation"?

Reply:
✅